CONTRIBUTING.md

```diff
@@ -1,40 +1,24 @@
 **Table of Contents**
 
 - [Contributing guidelines](#contributing-guidelines)
-    - [Terms](#terms)
-    - [Certificate of Origin](#certificate-of-origin)
-    - [DCO Sign Off](#dco-sign-off)
-    - [Code of Conduct](#code-of-conduct)
-    - [Contributing a patch](#contributing-a-patch)
-    - [Issue and pull request management](#issue-and-pull-request-management)
+  - [Terms](#terms)
+  - [Certificate of Origin](#certificate-of-origin)
+  - [DCO Sign Off](#dco-sign-off)
+  - [Code of Conduct](#code-of-conduct)
+  - [Contributing a patch](#contributing-a-patch)
+  - [Issue and pull request management](#issue-and-pull-request-management)
+- [References](#references)
 
 # Contributing guidelines
 
-The ocm repo contains 4 core components:
+The ocm repo contains 5 core components:
 * registration
 * placement
 * work
 * registration-operator
+* addon-manager
 
-Before 0.11.0, the 4 components has independent repos, now we are going through [the task of consolidating code](https://github.com/open-cluster-management-io/OCM/issues/128) to merge all code into this repo to gain all kinds of benifits.
-
-We're contiously working on migrate code from `/staging` folder to outside. Finally, the `/staging` folder will be empty and removed.
-
-Here is a table of the migration status:
-
-| Component | Status |
-| --- | --- |
-| registration | staging |
-| placement | staging |
-| work | staging |
-| registration-operator | staging |
-
-Status:
-* staging: the component is having all code in `staging` folder.
-* mix: the component is having code in both `staging` and outside.
-* done: the component is having all code outside `staging` folder and done of the migration.
-
-If a component is in the `staging` status, you need to open project in the `/staging` folder and contribute there.
+Before 0.11.0, the 5 components has independent repos, now we are going through [the task of consolidating code](https://github.com/open-cluster-management-io/OCM/issues/128) to merge all code into this repo to gain all kinds of benifits.
```

Comment:
we have consolidate the code into ocm repo.

The task is done, so we should not it is going through.

Reply:
removed this line